api/graphql/makeModels.js

```diff
@@ -218,6 +218,12 @@ export default function makeModels (userId, isAdmin) {
             type: filter,
             sort: sortBy,
             topic
+          }),
+        betaAccessCode: c =>
```

Comment:
btw i would avoid propagating this "beta access code" nomenclature to any user-facing API, since it's not really beta anymore. since it's only ever used in the context of an invite link, it's appropriate only to surface the whole link as an attribute of a community.